ext/pcre/php_pcre_arginfo.h

```diff
@@ -63,3 +63,6 @@ ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_preg_last_error, 0, 0, IS_LONG, 0)
 ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_WITH_RETURN_TYPE_INFO_EX(arginfo_preg_last_error_msg, 0, 0, IS_STRING, 0)
+ZEND_END_ARG_INFO()
```

Comment:
This is a generated file, the function should be added to `ext/pcre/php_pcre.stub.php` instead.

Reply:
Okay, I've added the file. Despite this being a generated file, it's tracked by git with all other arginfo. Should I undo this change, or leave both files modified?

Reply:
The `_arginfo.h` file should be updated by calling `scripts/dev/gen_stub.php ext/pcre` (normally that happens automatically), and the result committed.

Reply:
Okay, done! Thanks!